pkg/utils/cache.go

```diff
@@ -11,6 +11,12 @@ type Cache struct {
 	FileCache   FileCache
 	OrbCache    OrbCache
 	DockerCache DockerCache
+	TokenCache  TokenCache
+}
+
+type TokenCache struct {
+	cacheMutex *sync.Mutex
+	Token      string
```

Comment:
change `Token` to `token` to make it and private, use the mutex and the function `SetToken`